test/services/network/translator/test_network_translator.py

```diff
@@ -6,7 +6,7 @@
 import pytest
 from hypothesis import given, HealthCheck, settings
 
-from database.sqlite.schema_utils import assume_non_blank_street_address_details
+from test.database.sqlite.schema_utils import assume_non_blank_street_address_details
```

Comment:
Good catch on the import. This is my pet peeve as the import works in the IDE without the `test.` so I constantly miss it.